daq/runner.py

```diff
@@ -166,6 +169,9 @@ def __init__(self, config):
         self._system_active = False
         logging_client = self.gcp.get_logging_client()
         self.daq_run_id = self._init_daq_run_id()
+        self._device_result_client = None
```

Comment:
little bit clumsy doing it this way, best to assign the variable (_device_result_client) only in one place. E.g., 
self._device_result_client = self._init_device_result_client(self.config.get('device_result_port'))

Reply:
updated